content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -278,8 +278,6 @@ pod usage is still within acceptable limits.
 
 ### Container resource metrics
 
-{{< feature-state for_k8s_version="v1.27" state="beta" >}}
```

Comment:
Could we mark it is stable since v1.30 like:

{{< feature-state for_k8s_version="v1.30" state="stable" >}}